packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -105,7 +122,25 @@ export class AtomStepsModal {
     e.stopImmediatePropagation()
     this.isOpen = false
     this.atomDidDismiss.emit(this.currentStep)
-    this.currentStep = 1
+    this.currentStep = this.customInitialStep ? this.customInitialStep : 1
```

Comment:
suggestion:

` this.currentStep = this.customInitialStep ?? 1`

Reply:
done at [f814758](https://github.com/juntossomosmais/atomium/pull/627/commits/f81475800acd02c38b0999983bb92aff8b013f53)